### PR TITLE
[AWIBOF-6879] Set timeout to 30 minutes for UNMOUNTARRAY

### PR DESCRIPTION
**Pull request by mjlee34**
Set timeout to 30 min for UNMOUNTARRAY command.

Failures in CI tests are often because
UNMOUNTARRAY command exceptionally takes long time.

To workaround this problem, this commit sets the timeout for the command
to 30 minutes.

However, the timeout needs to be set based on a systemical approach
rather than a temporal constant.

### CODE REVIEW

**Review by ringopow — DISMISSED** [review]

File: doc/guides/cli/poseidonos-cli.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ poseidonos-cli [flags]
       --json-res         Print response in JSON form.
       --node string      Name of the node to send this command. When both --ip and this flag are specified, this flag is applied only.
       --port string      Set the port number to PoseidonOS for this command. (default "18716")
-      --timeout uint32   Timeout for this command in seconds. (default 180)
+      --timeout uint32   Timeout for this command in seconds. (Note: array mount command has 30 minutes timeout. (default 180)
```
Comment (hsung-yang) [inline doc/guides/cli/poseidonos-cli.md:34]: 추가된 안내문구의 여는괄호와 매칭되는 닫는 괄호가 없는 것 같습니다

Comment (mjlee34) [reply doc/guides/cli/poseidonos-cli.md:34]: 아, 꼼꼼히 봐주셔서 감사합니다.

**Review by junghyun-34 — DISMISSED** [review]
LGTM

**Review by hsung-yang — APPROVED** [review]
LGTM

**Review by ringopow — APPROVED** [review]

**Pull request merged**